docs/examples/101/databricks-workspace/main.bicep

```diff
@@ -18,8 +24,13 @@ resource ws 'Microsoft.Databricks/workspaces@2018-04-01' = {
   }
   properties: {
     // TODO: improve once we have scoping functions
```

Comment:
Is this TODO still relevant?

Reply:
Yes, I think the value of `managedResourceGroupId` should come from an `existing` reference. 

```bicep

resource managedRg 'Microsoft.Resources/resourceGroups@2021-04-01' existing = {
  name: managedResourceGroupName
}

...
managedResourceGroupId: managedRg.id
```

Reply:
thanks!